spec/lib/account_spec.rb

```diff
@@ -0,0 +1,101 @@
+describe BankAccount::Account do
+
+  describe '.new(id, initial_balance)' do
+    it 'responds' do
+      expect(BankAccount::Account).to respond_to :new
+    end
+
+    context 'when instantiating an accepted account' do
+      before :all do
+        @my_account = BankAccount::Account.new(1,125)
+      end
+
+      it 'is part of Account class' do
+        expect(@my_account).to be_a BankAccount::Account
+      end
+
+      it 'has an id' do
+        expect(@my_account.id).to eq(1)
+      end
+
+      it 'has a balance' do
+        expect(@my_account.balance).to eq(125)
+      end
+
+      it 'has an initial_balance' do
+        expect(@my_account.initial_balance).to eq(125)
+      end
+    end
+
+    context 'when a negative account balance is passed in' do
+      it 'raises an ArgumentError' do
+        expect { BankAccount::Account.new(1, -1) }.to raise_error(ArgumentError)
+      end
+    end
+  end
```

Comment:
I wasn't sure what the best practice is here for that.... is it good to just have it pass on any error so that if someone decides to change the error message it doesn't break the test, or is it better to include the message so that we can check to make sure that they are getting the _correct_ error? Or should there be different tests for these?